### ISSUE TITLE
namecheap support

### ISSUE THREAD

**Issue by hrvoj3e** [nickbabcock/dness#123]
```
[[domains]]
type = "namecheap"
domain = "deadbeef"
zone = "example2.com"
records = [ "mysubdomain" ]
```

Would be:
`https://dynamicdns.park-your-domain.com/update?host=mysubdomain&domain=example2.com&password=deadbeef&ip=[your_ip]`

Docs:
`https://dynamicdns.park-your-domain.com/update?host=[host]&domain=[domain_name]&password=[ddns_password]&ip=[your_ip]`

IP Address = optional value. If you don't specify any IP, the IP from which you are accessing this URL will be set for the domain.

https://www.namecheap.com/support/knowledgebase/article.aspx/29/11/how-do-i-use-a-browser-to-dynamically-update-the-hosts-ip

**Comment by nickbabcock** [comment]
I did not know that was possible! Yeah I'll look to implement this over the coming weeks.

**Comment by nickbabcock** [comment]
This feature is done and committed on master. It works with the namecheap domains I've registered. The last thing I need to do is write some docs and push a release. I hope to have this done in the next 24 hours.

**Comment by nickbabcock** [comment]
Done and included in the v0.2.1 release
